### PR TITLE
Create "Goodbye Phonegap" tweet.

### PR DESCRIPTION
**Pull request by NiklasMerz**
Tweet new blog post: https://github.com/apache/cordova-docs/pull/1114/files

**Wait until it's deployed**

### CODE REVIEW

**Comment by NiklasMerz** [comment]
Preview: https://github.com/apache/cordova/runs/986913224

File: tweets/goodbye-phonegap.tweet [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The contributors of Apache Cordova would like thank the @phonegap team and say goodbye. The core of PhoneGap is still alive as Apache Cordova.
+
+https://cordova.apache.org/announcements/2020-08-14-goodbye-phonegap.html
```
Comment (erisu) [inline tweets/goodbye-phonegap.tweet:3]:
```suggestion
https://cordova.apache.org/announcements/2020/08/14/goodbye-phonegap.html
```

**Comment by erisu** [comment]
Updated Preview: https://github.com/apache/cordova/runs/987529991

File: tweets/goodbye-phonegap.tweet [hunk 2]
```diff
@@ -0,0 +1,3 @@
+The contributors of Apache Cordova would like thank the @phonegap team and say goodbye. The core of PhoneGap is still alive as Apache Cordova.
```
Comment (erisu) [inline tweets/goodbye-phonegap.tweet:1]:
```suggestion
The contributors of Apache Cordova would like to thank the @phonegap team and say goodbye. The core of PhoneGap is still alive as Apache Cordova.
```

**Pull request merged**